strapi/strapi
# fix: making api tokens creation work

### What does it do?

- Makes sure that the lifespan is a number not a string
- Makes sure the user gets navigated to the correct page after successful creation
- Add e2e tests to for creating API tokens

### Why is it needed?

- Creating API tokens was broken

### How to test it?

- Attempt to create an API token, it should work and you should be able to copy the token

### Related issue(s)/PR(s)

fix #19994
